config/config.go

```diff
@@ -92,7 +97,10 @@ func OverrideConfigFromOptions(c *Config) {
 func OverrideConfigFromEnvVars(c *Config) error {
 	overrideConfigLogEnvVars(c)
 	overrideConfigCheckEnvVars(c)
-	overrideCommConfigEnvVars(c)
+	if err := overrideCommConfigEnvVars(c); err != nil {
+		return err
+	}
+	c.VcsRevision = getVcsRevision()
```

Comment:
VCS info has nothing to do with env vars, right? It doesn't seem to belong to this function (`OverrideConfigFromEnvVars`).

Reply:
right 537475f